src/pages/index.js

```diff
@@ -27,126 +57,880 @@ const Page = styled.div`
-        alt: "page-index-sections-individuals-image-alt",
-      },
-      title: "page-index-sections-individuals-title",
-      desc: "page-index-sections-individuals-desc",
-      link: {
-        text: "page-index-sections-individuals-link-text",
-        to: "/what-is-ethereum/",
-      },
+      image: data.ethfixed.childImageSharp.fixed,
+      title: translateMessageId("page-index-get-started-eth-title", intl),
+      description: translateMessageId(
+        "page-index-get-started-eth-description",
+        intl
+      ),
+      alt: translateMessageId("page-index-get-started-eth-image-alt", intl),
+      to: "/get-eth/",
     },
     {
-      img: {
-        src: data.developers,
-        alt: "page-index-sections-developers-image-alt",
-      },
-      title: "page-index-sections-developers-title",
-      desc: "page-index-sections-developers-desc",
-      link: {
-        text: "page-index-sections-developers-link-text",
-        to: "/developers/",
-      },
+      image: data.dogefixed.childImageSharp.fixed,
+      title: translateMessageId("page-index-get-started-dapps-title", intl),
+      description: translateMessageId(
+        "page-index-get-started-dapps-description",
+        intl
+      ),
+      alt: translateMessageId("page-index-get-started-dapps-image-alt", intl),
+      to: "/dapps/",
+    },
+    {
+      image: data.devfixed.childImageSharp.fixed,
+      title: translateMessageId("page-index-get-started-devs-title", intl),
+      description: translateMessageId(
+        "page-index-get-started-devs-description",
+        intl
+      ),
+      alt: translateMessageId("page-index-get-started-devs-image-alt", intl),
+      to: "/developers/",
+    },
+  ]
+
+  const touts = [
+    {
+      image: data.docking.childImageSharp.fixed,
+      alt: translateMessageId("page-index-tout-eth2-image-alt", intl),
+      title: translateMessageId("page-index-tout-eth2-title", intl),
+      description: translateMessageId("page-index-tout-eth2-description", intl),
+      to: "/eth2/",
+    },
+    {
+      image: data.infrastructurefixed.childImageSharp.fixed,
+      alt: translateMessageId("page-index-tout-enterprise-image-alt", intl),
+      title: translateMessageId("page-index-tout-enterprise-title", intl),
+      description: translateMessageId(
+        "page-index-tout-enterprise-description",
+        intl
+      ),
+      to: "/enterprise/",
+    },
+    {
+      image: data.enterprise.childImageSharp.fixed,
+      alt: translateMessageId("page-index-tout-community-image-alt", intl),
+      title: translateMessageId("page-index-tout-community-title", intl),
+      description: translateMessageId(
+        "page-index-tout-community-description",
+        intl
+      ),
+      to: "/enterprise/",
+    },
+  ]
+
+  const features = [
+    {
+      title: price,
+      description: translateMessageId(
+        "page-index-network-stats-eth-price-description",
+        intl
+      ),
+      emoji: ":money_with_wings:",
```

Comment:
@ryancreatescopy can we delete these emoji fields? Or are we planning to incorporate these into the metric cards somehow? cc @wackerow